.github/workflows/stylish.yml

```diff
@@ -5,34 +5,52 @@ on:
   workflow_dispatch:
 
 jobs:
-  stylish:
-    name: "black & flake8 & rpmlint"
+  check-spec:
+    name: rpmlint
     runs-on: ubuntu-latest
     container:
       image: fedora:latest
 
     steps:
-    - name: Base setup
-      run: |
-        dnf --setopt install_weak_deps=False install -y \
-            git-core \
-            python3-flake8 \
-            python3-pip \
-            rpmlint
-
-    - uses: actions/checkout@v3
-
-    - uses: psf/black@stable
-      with:
-        version: "22.3.0"
-
-    - name: Setup flake8 annotations
-      uses: rbialon/flake8-annotations@v1
-
-    - name: Run flake8
-      run: |
-        flake8
-
-    - name: Run rpmlint
-      run: |
-        rpmlint subscription-manager.spec
+      - name: Install packages
+        run: |
+          dnf --setopt instal_week_deps=False install -y git-core rpmlint
+
+      - uses: actions/checkout@v3
+
+      - name: Run rpmlint
+        run: |
+          rpmlint subscription-manager.spec
+
+  check-python:
+    name: "black & flake8"
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        # By using '-dev' suffix, Actions picks up the latest release,
+        # including alpha, beta and rc versions.
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11-dev"]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: "pip"
+
+      - name: Install packages
+        run: |
+          python3 -m pip install --upgrade pip wheel
+          python3 -m pip install -r test-requirements.txt
+
+      - name: Run Black
+        run: |
+          black . --diff
+
+      - name: Run flake8
+        run: |
+          flake8 . --count --statistics
```

Comment:
hmm a simple `flake8` is enough, isn't it?

Reply:
it seems that both `count` and `statistics` can be specified in the configuration file, so in case I'd prefer that (so it works in any case); in that case, with the options added in the config, the path is not needed anymore, so the simple `flake8` invocation is still fine again :)